### DISCUSSION TITLE
Set default to "no-access" for all new collections instead "view and edit"

### DISCUSSION THREAD

**Discussion by ReessKennedy** [outline/outline#3921] [Ideas]
TL;DR: It's annoying to have go in and change the permissions of each collection one at a time. It'd be great to set the default permissions for new collections in advance. 

Longer: my default is to write lots of private notes in new collections and only share a few.  It's, therefor, really annoying and prone to error when I realize that all these new collections I created will be visible unless I go in and change the permissions for each collection. 

Perhaps I missed some instruction but, maybe, a setting that allows you to change the permissions of a collection in advance would help.

**Comment by tommoor** [comment]
Hi @ReessKennedy – the option to choose collection visibility is there at creation, we do need to refine the design on this flow though, it should probably be clearer than a simple dropdown.

**Reply by ReessKennedy** [reply]
Thanks, @tommoor ... appreciate all your leadership and work here.

**Comment by cmer** [comment]
I fully agree. I also want all new collections to be "No Access" by default. We too, create many private collections, and it's very risky that sensitive information "leaks" because someone forgot to change the permissions at creation. A sensible secure default would be much better for us. Thanks!

**Comment by cmer** [comment]
Pretty, please, @tommoor :)
